src/SFA.DAS.RoATPService.Application/commands/DeleteOrganisationShortCourseTypes/DeleteOrganisationShortCourseTypesValidator.cs

```diff
@@ -0,0 +1,26 @@
+﻿using FluentValidation;
+using SFA.DAS.RoATPService.Domain.Repositories;
+
+namespace SFA.DAS.RoATPService.Application.commands.DeleteOrganisationShortCourseTypes;
+public class DeleteOrganisationShortCourseTypesValidator : AbstractValidator<DeleteOrganisationShortCourseTypesCommand>
+{
+    public const string InvalidUkprnMessage = "Ukprn does not exist";
+    public const string UkprnIsRequiredMessage = "Ukprn must not be empty";
+    public const string RequestingUserIdIsRequiredMessage = "RequestingUserId must not be empty";
+
+    public DeleteOrganisationShortCourseTypesValidator(IOrganisationsRepository organisationRepository)
+    {
+        RuleFor(c => c.Ukprn)
+            .GreaterThan(0)
+            .WithMessage(UkprnIsRequiredMessage)
+            .MustAsync(async (id, token) =>
+            {
+                var org = await organisationRepository.GetOrganisationByUkprn(id, token);
+                return org != null;
+            })
+            .WithMessage(InvalidUkprnMessage);
+        RuleFor(c => c.RequestingUserId)
```

Comment:
This rule is redundant as the action will not be triggered at all if the header is missing due to RequiredHeader filter